networkx/drawing/layout.py

```diff
@@ -364,8 +408,6 @@ def spring_layout(
     as repelling objects, sometimes called an anti-gravity force.
     Simulation continues until the positions are close to an equilibrium.
 
-    There are some hard-coded values: minimal distance between
-    nodes (0.01) and "temperature" of 0.1 to ensure nodes don't fly away.
```

Comment:
The minimal node distance is no longer needed.
It was primarily there to prevent a division by zero, which is currently handled in a different way.

The temperature never actually was 0.1, it was 0.1* the maximum extent of the positions of the nodes.
Now it is `0.1 * max(dom_size)`.
If you want to add that one again we can do so.